.github/scripts/dapr_bot.js

```diff
@@ -145,35 +142,14 @@ async function cmdAssign(github, issue, username, isFromPulls) {
         return;
     }
 
-    await github.issues.addAssignees({
+    await github.rest.issues.addAssignees({
         owner: issue.owner,
         repo: issue.repo,
         issue_number: issue.number,
         assignees: [username],
     });
 }
 
-/**
- * Comment a funny joke.
```

Comment:
You cannot remove this without discussing with maintainers first. That's not ok.

Now I happen to know that this isn't being used at all and I agree that we can remove it, but please run this kind of change by maintainers first in the future.